hack/examples/bases/aws/cluster/kustomization.yaml.tmpl

```diff
@@ -5,7 +5,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/${CAPA_VERSION}/cluster-template-simple-clusterclass.yaml
+- https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v2.9.1/cluster-template-simple-clusterclass.yaml
```

Comment:
Pretty high chance we will forget to revert this back.
How about we change the URL so that in the future when we go back to upstream CAPA it will fail on a missing tag?

Reply:
We don't have a 2.9.1 release in our fork

Reply:
:thinking: lets give it a try. We can do a release of 2.9.1 although im not sure if/how that will work. Can you help out with that?

Reply:
Can't we just use `v2.10.0-ncn.0` here? The artifacts are available via the GH release.

Reply:
☝️ yes thats what I meant too, to use the fork with `v2.10.0-ncn.0`

Reply:
can you be more clear in the future?